packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -89,6 +89,11 @@ class MediaQueryData {
   /// system UI (such as the system notification area), or or physical
   /// intrusions in the display (e.g. overscan regions on television screens or
   /// phone sensor housings).
+  ///
+  /// If you consumed this padding (e.g. by building a widget that envelops or
+  /// accounts for this padding in its layout), you should remove this padding
```

Comment:
This is *mostly* true, but there are cases, such as translucent widgets, where you both want to account for the padding and allow children to account for it. Maybe make the language less absolute or add something like 'if your widget accounts for this padding such that child widgets do not need to' (though ideally with better words than those...)

Reply:
Added more comments